docs/docs/jsx-in-depth.md

```diff
@@ -196,13 +196,7 @@ function NumberDescriber(props) {
 }
 ```
 
-Another option is to use ternary statements. For example, using the same function as above:
-
-```js{3-7}
-function NumberDescriber(props) {
-  return <div>{props.number} is an {props.number % 2 == 0 ? <strong>even</strong> : <i>odd</i>}
-}
-```
+You can learn more about [conditional rendering](https://facebook.github.io/react/docs/conditional-rendering.html) and [loops](https://facebook.github.io/react/docs/lists-and-keys.html) in the corresponding sections.
```

Comment:
Minor nit: please remove `https://facebook.github.io/react` prefix from the links so that they work locally.